ToolBox/IEX/IEXDataQueueHandler.cs

```diff
@@ -257,6 +257,16 @@ public void SetJob(LiveNodePacket job)
         {
         }
 
+        /// <summary>
+        /// Checks if this brokerage supports the specified symbol
+        /// </summary>
+        /// <param name="symbol">The symbol</param>
+        /// <returns>returns true if brokerage supports the specified symbol; otherwise false</returns>
+        private static bool CanSubscribe(Symbol symbol)
+        {
+            return symbol.SecurityType == SecurityType.Equity;
```

Comment:
Believe IEX does have support for options and futures, let's double check the exact security type supported list 👍